code/modules/mob/living/carbon/human/life/handle_mutations_and_radiation.dm

```diff
@@ -73,3 +97,201 @@
+
+		6 'gates'
+		100
+			Vomiting - Early on symptom, if rad_tick and radiation is above a certain level
+			Nosebeeds - Early onset of the blood vessels breaking down
+		200
+			Internal hemorrhaging - Lower rad tick has lower probability, but this ramps up as it goes along
+		300
+			Organ Damage - A little later on, again lower rad tick has lower prob
+		400
+			Minor limb mutation - We're talking a decently large exposure causing this, somewhat rare
+			Blindness - About the same chance as limb mutation
+			Clone damage - To emulate DNA breakdown
+		500
+			Necrosis - After Limb mutation
+			Glowing slightly green - Extremely rare, practically terminal, glowing increases with radiation level
+		600
+			Ghoulification if extremely lucky - We're talking 1 in 1000
+
+		For anything affecting internal or external organs, if it's robotic, exempt them.
+		*/
+
+		//For symptoms that you want to scale off of a minimum radiation amount
+		var/minor_rad_multiplier = max(1, radiation/20)
+		var/rad_multiplier = max(1, radiation/50)
+		var/major_rad_multiplier = max(1, radiation/70)
+		var/extreme_rad_multiplier = max(1, radiation/100)
+
+		if(rad_tick > RADDOSELIGHT)
+			if(prob(5*rad_multiplier))
+				//Vomit
+				vomit()
+			if(prob(5*minor_rad_multiplier))
+				//Nosebleed
+				if(prob(15))
+					to_chat(src, "<span class = 'danger'>Your nose starts bleeding!</span>")
+				drip(1)
+			if(prob(5*major_rad_multiplier))
+				//Hallucination
+				hallucination += rand(1,5)*minor_rad_multiplier
+		if(rad_tick > RADDOSEMINOR)
+			if(prob(2*major_rad_multiplier))
+				//Internal hemorrhaging
+				var/list/limbs_to_bleed = list()
+				for(var/datum/organ/external/E in organs)
+					if(E.is_robotic())
+						continue
+					limbs_to_bleed.Add(E)
+				if(limbs_to_bleed.len)
+					var/datum/organ/external/victim = pick(limbs_to_bleed)
+					if(prob(35))
+						to_chat(src, "<span class = 'danger'>You feel something tear in your [victim.display_name]</span>")
+					var/datum/wound/internal_bleeding/I = new (1*minor_rad_multiplier)
+					victim.wounds += I
+		if(rad_tick > RADDOSEADVANCED)
+			if(prob(5*rad_multiplier))
+				//Organ damage
+				var/list/organ_to_damage = list()
+				for(var/datum/organ/internal/I in internal_organs)
+					if(I.robotic)
+						continue
+					organ_to_damage.Add(I)
+				if(organ_to_damage.len)
+					var/datum/organ/internal/victim = pick(organ_to_damage)
+					victim.damage += rand(1,5)*rad_multiplier
+			if(prob(0.5*major_rad_multiplier))
+				//Become uncloneable
+				if(!(M_NOCLONE in mutations))
+					to_chat(src, "<span class = 'blob'>You feel something twist and break.</span>")
+					mutations |= M_NOCLONE
+		if(rad_tick > RADDOSECRITICAL)
+			if(prob(2*major_rad_multiplier))
+				//Minor limb mutation
+				var/list/datum/organ/external/candidates = list()
+				for (var/datum/organ/external/O in organs)
+					if(!(O.status & ORGAN_MUTATED|ORGAN_ROBOT))
```

Comment:
This may mutate amputated limbs because heh organ code

if(O.is_usable() && O.is_organic())

is_usable() checks if the organ is destroyed or mutated